src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -253,6 +253,9 @@ protected void VerifyParameters(params (string name, string documentation)[] par
             }
         }
 
+        protected void VerifySignatureHelpIsActive(bool expected) =>
```

Comment:
I know you're following the existing convention in this file, but the one-line Verify* methods are a bit silly. I'd rather see `Assert.True(Editor.IsSignatureHelpActive())`/`Assert.False(Editor.IsSignatureHelpActive())` than `VerifySignatureHelpIsActive(true)`/`VerifySignatureHelpIsActive(false)`.